README.md

```diff
@@ -100,42 +99,57 @@ var Cloudant = require('@cloudant/cloudant');
 // Initialize Cloudant with settings from .env
 var username = process.env.cloudant_username || "nodejs";
 var password = process.env.cloudant_password;
-var cloudant = Cloudant({account:username, password:password});
+var cloudant = Cloudant({ account:username, password:password });
 
-// Remove any existing database called "alice".
-cloudant.db.destroy('alice', function(err) {
+// Using the async/await style.
 
-  // Create a new "alice" database.
-  cloudant.db.create('alice', function() {
+async function asyncCall() {
+  await cloudant.db.create('alice');
+  return await cloudant.use('alice').insert({ happy: true }, 'rabbit');
+}
 
-    // Specify the database we are going to use (alice)...
-    var alice = cloudant.db.use('alice')
+asyncCall().then((data) => {
```

Comment:
I'll rm that legacy/readme-examples.js. It's incomplete. I'll add a new test file containing all code from README.md.

Reply:
@smithsz as an FYI there were some necessary changes to that legacy code that I did recently to support parallel running (i.e. using a unique database name instead of `alice` or `test` or whatever. Pretty much just adding constants for the DB names instead of using the same string that was used in the examples.
```
const alice = `nodejs_cloudant_test_${uuid()}`
const my_db = `nodejs_cloudant_test_${uuid()}`
```